package/MDAnalysis/coordinates/XYZ.py

```diff
@@ -148,10 +147,8 @@ def __init__(self, filename, n_atoms=None, atoms=None, convert_units=None,
         """
```

Comment:
This is on the Writer, so this being true would convert inputs to the native units ... for xyz?  I'm not sure if XYZ has a native unit? So I'm not going to break anything here, but I'm not sure I want to document the fact that this option exists?

Reply:
XYZ units are _generally_ in angstroms and that's how most visualization packages interpret them (VMD for example).

Reply:
So MDA does seem to assume `ps/Angstrom` on read, and the default writer also assume those units on the conversion. I guess the only case I could think of where this might be useful, is if for some reason you created a `Universe` with non-base units, and then wanted to use the writer to have a non-`ps/Angstrom` write. With the removal of flags, I'm not sure how likely that would be to happen though...